layouts/_default/home.html

```diff
@@ -192,18 +191,30 @@ <h2>{{ T "IndexPage.Install.Heading" }}</h2>
 					<span class="iconify-inline" data-icon="simple-icons:microsoftedge"></span> {{ T "Global.EdgeShort" }}
 				</a>
 			</p>
-			<p>
+			<p class="mb-4">
 				{{- .Scratch.Add "open_tag_2" "<a href='" }}
 				{{- .Scratch.Add "open_tag_2" ( "/docs/getting-started/installing#from-source" | relLangURL ) }}
 				{{- .Scratch.Add "open_tag_2" "' target='_blank' rel='noopener'>" }}
 				{{- T "IndexPage.Install.InstallButton.Other" ( dict "OpenTag1" "<a href='https://github.com/ScratchAddons/ScratchAddons/releases' target='_blank' rel='noopener'>" "CloseTag1" "</a>" "OpenTag2" ( .Scratch.Get "open_tag_2" ) "CloseTag2" "</a>" ) | safeHTML }}
 			</p>
+			<div class="">
+				<p id="total-users" style="font-size: 3rem; font-weight: 500; line-height: 3rem;" class="odometer mb-0"></p>
+				<p style="font-size: 1.75rem; font-weight: 500;">{{ T "IndexPage.Users" }}</p>
+			</div>
 		</div>
-		<div class="col-12 col-lg-auto ml-lg-3 d-flex flex-column justify-content-center">
-			<p id="total-users" style="font-size: 3rem; font-weight: 500; line-height: 3rem;" class="odometer mb-0"></p>
-			<p style="font-size: 1.75rem; font-weight: 500;">{{ T "IndexPage.Users" }}</p>
+		<div class="col-12 col-lg-6 d-md-flex flex-column justify-content-center">
+			<h2>Try</h2>
+			<p>Try some of the editor addons on TurboWarp before installing.</p>
+			<p>
+				<a class="btn btn-success dark-btn-white btn-lg mb-4" href="https://turbowarp.org/addons" target="_blank" rel="noopener">

```

Comment:
I think we have no better option that linking to `turbowarp.org/editor` for now 🤔

Reply:
I propose https://turbowarp.org/discover-addons, which just goes to the editor right now, but means I can change it later without bothering anyone here (it may end up making more sense to show people a documentation page than dropping them straight into the editor, for example)

Reply:
I have to remind you the context of this button. The purpose of this button is to let people try the experience of some SA addons on TurboWarp, so something like a documentation page doesn't fit well, although that's just on me.

Reply:
Just an example. Most likely it will just go to the editor but with some extra nags to remind people the addon button exists